source/system/system.go

```diff
@@ -153,6 +162,16 @@ func splitVersion(version string) map[string]string {
 	return components
 }
 
+// Read /sys/devices/virtual/dmi/id/sys_vendor
+func getVendor() (string, error) {
+	vendor, err := os.ReadFile("/sys/devices/virtual/dmi/id/sys_vendor")
```

Comment:
Here we can replace `/sys` with the variable `hostpath.SysfsDir`

Reply:
@AhmedGrati
Updated, thanks!